src/Init/Data/BitVec/BasicAux.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joe Hendrix, Wojciech Nawrocki, Leonardo de Moura, Mario Carneiro, Alex Keizer, Harun Khan, Abdalrhman M Mohamed
+-/
+prelude
+import Init.Data.Fin.Basic
+
```

Comment:
Could you add a module doc to this file explaining why the `Basic`/`BasicAux` split is needed in this case? Probably more or less the contents of https://lean-fro.zulipchat.com/#narrow/stream/398861-general/topic/Reworking.20import.20structure.20in.20Init/near/467753170